Snowflake.Data/Client/SnowflakeDbConnectionPool.cs

```diff
@@ -17,6 +18,7 @@ class SnowflakeDbConnectionPool
         private static long timeout;
         private const int MAX_POOL_SIZE = 10;
         private const long TIMEOUT = 3600;
+        private static bool pooling = false;
```

Comment:
Please set pooling back to the original default value. I am assuming that it was enabled by default. Any deviation from original default values constitute a behavior change.